multisrc/overrides/madara/randomscan/src/LunaToon.kt

```diff
@@ -7,9 +7,9 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class RandomScan : Madara(
+class LunaToon : Madara(
     "Random Scan",
```

Comment:
change name here like you did in the generator, also need to override the id as beer asked earlier.

Reply:
Done, I changed the name of the file to LuraToon.kt and the class to LuraToon, according to MadaraGeneretor in "className = "LuraToon""

Reply:
but anything?, before making the commit?

Reply:
> but anything?, before making the commit?

as AwkwardPeak said, you need to:
- Change the source name: `"Random Scan"` -> `"Lura Toon"`
- Override the ID: find the old source ID [here](https://raw.githubusercontent.com/keiyoushi/extensions/repo/index.json) and see how to override the ID [here](https://github.com/search?q=repo%3Akeiyoushi%2Fextensions-source+%22override+val+id%22&type=code).